src/serac/physics/contact/contact_data.cpp

```diff
@@ -323,10 +322,15 @@ void ContactData::updateDofOffsets() const
   }
   if (HYPRE_AssumedPartitionCheck()) {
     auto total_dofs = global_pressure_dof_offsets_[global_pressure_dof_offsets_.Size() - 1];
-    global_pressure_dof_offsets_.SetSize(3);
+    if (mesh_.GetNRanks() < 3) {
+      global_pressure_dof_offsets_.SetSize(3);
```

Comment:
Thanks for bringing attention to his, it looks like this should be 2 instead of 3.  This came up in single rank testing: the size of global_pressure_dof_offsets_ is two in this case and the code tries to access the third entry.  I added a comment to clarify what's going on here.

Reply:
I don't see the change. Do you need to push?

Reply:
Not sure why github isn't showing the changes, but it looks like they're in the branch?  https://github.com/LLNL/serac/blob/feature/chin23/new_contact_examples/src/serac/physics/contact/contact_data.cpp#L326-L335